cfpb/sbl-frontend
# feat(Resolve Errors): Split URL paths between syntax and logic errors

closes #845 

## Changes

- feat(Resolve Errors): Split URL paths between syntax and logic errors

## How to test this PR

1. Upload a logic errors csv, continue to next step - verify ~~`/errors-2`~~ `/errors-logic`
2. Upload a syntax errors csv, continue to next step- verify ~~`/errors-1`~~ `/errors-syntax`
3. Upload no errors with warnings csv, continue to next step - verify ~~`/errors`~~ `/`
4. Upload no errors with no warnings csv, continue to next step - verify ~~`/errors`~~ `/`

- no syntax errors but has logic errors (and vice versa) -- should redirect properly.
- Try to manually navigate (e.g. type in `http://localhost:8899/filing/2024/123456789TESTBANK456/errors/errors-logic`) and check if the redirect is correct
- Try using the browser's Back and Forward arrows
- Use the Go Back and Continue buttons
